test/extended/prometheus/alerts.go

```diff
@@ -0,0 +1,94 @@
+package prometheus
+
+import (
+	"time"
+
+	g "github.com/onsi/ginkgo"
+
+	v1 "k8s.io/api/core/v1"
+	policyv1beta1 "k8s.io/api/policy/v1beta1"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	intstr "k8s.io/apimachinery/pkg/util/intstr"
+	"k8s.io/apimachinery/pkg/util/wait"
+
+	clientset "k8s.io/client-go/kubernetes"
+	e2e "k8s.io/kubernetes/test/e2e/framework"
+	"k8s.io/kubernetes/test/e2e/framework/pod"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+)
+
+var _ = g.Describe("[Feature:PodDisruptionBudgetAtLimitAlert][Conformance] Prometheus", func() {
```

Comment:
This is so if/when this test fails cluster-kube-controller-manager-operator (API/Workloads) peeps might get the bug/issue rather than monitoring folks.